source/mongodump.txt

```diff
@@ -264,6 +264,13 @@ Behavior
    the :dbtools:`--archive </mongodump/#std-option-mongodump.--archive>` 
    option.  
 
+Using ``mongodump`` Without an ``authSource``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When :urioption:`authSource` is not specified in the MongoDB URI, the database name provided 
+with :option:`--db` is used to authenticate your ``mongodump`` session
```

Comment:
[nit]

Maybe "the database name specified in the  :option:`--db` option is used..."

Reply:
Updated.